src/style.css

```diff
@@ -0,0 +1,60 @@
+@font-face {
+  font-family: Avenir-Book;
+  src: url("./fonts/Avenir-Book.ttf"), format("ttf");
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url("./fonts/Avenir-Heavy.ttf"), format("ttf");
+}
+
+html {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+}
+
+.form-set {
+  margin-bottom: 20px;
+  width: 100%;
+}
+
+.form-field::placeholder {
+  font-weight: 300;
+}
+
+.form-field {
+  display: block;
+  width: 100%;
+  margin: 0 auto;
+  border-radius: 4px;
+  text-indent: 62px;
+  font-size: 16px;
+  font-weight: 600;
+  line-height: 22px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  background-position: 25px 50%;
+  cursor: pointer;
+}
+
+.form-field:hover {
+  box-shadow: 0 3px 8px 0 #3d4e6133;
+}
+
+.form-field:focus {
+  background: url("./images/Search.svg"), linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  background-repeat: no-repeat;
```

Comment:
Add ::before element for input and set for him search image as background or correct background position (now image changes position on focus state)